pyproject.toml

```diff
@@ -42,7 +42,7 @@ packages = ["maturin"]
 bindings = "bin"
 
 [tool.black]
-target_version = ['py37']
+target-version = ['py37']
```

Comment:
my IDE said that it was the wrong key and in the black repo the key `target-version` is used

https://github.com/psf/black/blob/96ca1b6be3e543618dbcf791d1c143e7119cb139/pyproject.toml#L10